dirsrvtests/tests/suites/state/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""
+   :Requirement: 389-ds-base: Replication
```

Comment:
Adding this under replication as not sure should I create a new one or not

Reply:
I think the requirement here could be 'operational attributes'